packages/admin/src/services/SuperAdminService.ts

```diff
@@ -140,3 +141,27 @@ export const createDepartmentInformation = async (
     body,
     axiosUserServiceConfig(userToken)
   );
+
+export const filterUsers = async (
+  pagination: Pagination,
+  filterData: SuperAdminDashboardFilterData,
+  userToken: string
+) => {
+  const params = {
+    pagination,
+    roles: filterData.roles.join(','),
+    departments: filterData.departments.join(','),
+    searchTerm: filterData.searchTerm,
+    clearAllFilters: filterData.clearAllFilters,
+  };
```

Comment:
we should be able to combine this with `getSuperAdminDashboard` and just make `filterData` optional?